README.md

```diff
@@ -2,79 +2,79 @@
 
 ## Introduction
 
-So you want to contribute to the Rocky Linux documentation, but aren't sure how to go about that? You've come to the right place. We're excited to have you on board with us. 
+So you want to contribute to Rocky Linux documentation but aren't sure how to go about that? You came to the right place. We are privileged to have you on board with us. 
 
-One of our big goals for Rocky is to have not just good, but GREAT documentation, and we can't do it alone. This document will get you started, and will (hopefully) allay any worries you might have about doing so. We're always here to help, too, so feel free to ask questions and join in the conversation.
+One of our main goals for Rocky is to have not just good but GREAT documentation, and we cannot do it alone. This document will get you started and will (hopefully) ease any worries you might have about doing so. We will always be here to help, so feel free to ask questions and join in the conversation.
 
 ## Documentation Subject Matter
 
-If you can dream it — or better yet, have done it before — then your subject matter is probably important to the Rocky Linux documentation project. Once a build is finally out there, then we expect that documentation will really start to explode, but we want to be ready with some things already documented out of the gate. 
+If you can dream it — or better yet, have done it before — then your subject matter is probably essential to the Rocky Linux documentation project. Once a build is finally out there, we expect that documentation will start to explode, but we want to be ready with some things already documented out of the gate. 
 
-This is where you come in — both now and in the future. If you have experience with anything Linux-related, and can translate that to Rocky Linux, then we want you to start writing!
+Where do you come in — both now and in the future? If you have experience with anything Linux-related and can translate that to Rocky Linux, we would like you to start writing!
 
 ## Where to Start
 
-While you can create documentation in any format, the preferred format is Markdown. You can easily create documents in Markdown, plus Markdown is super-easy to learn. The best way to get started with Markdown is to have a Markdown editor installed and read the tutorial. You can create Markdown files with any text editor, but most Markdown editors allow you to preview what you've input already, so they can be very helpful. 
+While you can create documentation in any format, the preferred format is Markdown. You can easily create documents in Markdown, plus Markdown is simple to learn. The best way to get started with Markdown is to have a Markdown editor installed and read the tutorial. You can create Markdown files with any text editor, but most Markdown editors allow you to preview what you've input already so that they can be beneficial. 
```

Comment:
"what you've input already" should be "what you have inputted already" I missed this